doc/metrics.md

```diff
@@ -310,9 +310,9 @@ async function myTask() {
     const executionTime = endTime - startTime
 
     httpServerDuration.record(executionTime, {
-      [SemanticAttributes.HTTP_METHOD]: 'POST',
-      [SemanticAttributes.HTTP_STATUS_CODE]: '200',
-      [SemanticAttributes.HTTP_SCHEME]: 'https',
+      [HTTP_METHOD]: 'POST',
+      [HTTP_STATUS_CODE]: '200',
+      [HTTP_SCHEME]: 'https',
```

Comment:
While I agree the documentation should demonstrate the best way to use these values, since it's not imported in the examples, it is now confusing where these are from.